distributed/scheduler.py

```diff
@@ -2007,8 +2006,8 @@ def cancel_key(self, key, client, retries=5, force=False):
             return
         if ts is None or not ts.who_wants:  # no key yet, lets try again in a moment
             if retries:
-                self.loop.add_future(
-                    gen.sleep(0.2), lambda _: self.cancel_key(key, client, retries - 1)
+                self.loop.call_later(
+                    0.2, lambda _: self.cancel_key(key, client, retries - 1)
                 )
```

Comment:
Fixed. There's no reason to manually call `gen.sleep` here, using `IOLoop.call_later` works just as well.